Documentation/network/kubernetes/configuration.rst

```diff
@@ -150,65 +144,59 @@ The CNI auto installation is performed as follows:
 1. The ``/etc/cni/net.d`` and ``/opt/cni/bin`` directories are mounted from the
    host filesystem into the pod where Cilium is running.
 
-2. The file ``/etc/cni/net.d/05-cilium.conf`` is written in case it does not
-   exist yet.
-
-3. The binary ``cilium-cni`` is installed to ``/opt/cni/bin``. Any existing
+2. The binary ``cilium-cni`` is installed to ``/opt/cni/bin``. Any existing
    binary with the name ``cilium-cni`` is overwritten.
 
-.. _install_cni:
-
-Manually installing CNI
------------------------
-
-This step is typically already included in all Kubernetes distributions or
-Kubernetes installers but can be performed manually:
-
-.. code-block:: shell-session
-
-    sudo mkdir -p /opt/cni
-    wget https://storage.googleapis.com/kubernetes-release/network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
-    sudo tar -xvf cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz -C /opt/cni
-    rm cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
+3. The file ``/etc/cni/net.d/05-cilium.conflist`` is written.
```

Comment:
Does this mean that `/etc/cni/net.d/05-cilium.conf` no longer exists? Asking because I can still observe several occurrences of `05-cilium.conf` (not `...conflist`) in the repo.

Is it worth a note in `Documentation/operations/upgrade.rst`?

Reply:
Very good observation on both points!

Reply:
Thanks!

I can still observe the following:

```
$ rg -w '05-cilium.conf'
[...]
Documentation/installation/cni-chaining-portmap.rst
53:``05-cilium.conflist`` and remove the standard ``05-cilium.conf``. The new

install/kubernetes/cilium/files/nodeinit/startup.bash
101:    echo -e "{\n\t"cniVersion": "0.3.1",\n\t"name": "cilium",\n\t"type": "cilium-cni"\n}" > /etc/cni/net.d/05-cilium.conf
```

Please update if relevant.

Not sure if you agreed on adding a note to `Documentation/operations/upgrade.rst` (I believe it would be a good thing), but I can't find one in the updated PR.

Reply:
Whoops, forgot to `git add` :-p

Reply:
The entry in `startup.bash` is a placeholder; there's no need to change it.